examples/time_frequency/plot_epochs_spectra.py

```diff
@@ -43,3 +50,18 @@
 
 # Now let's take a look at the spatial distributions of the psd.
 epochs.plot_psd_topomap(ch_type='grad', normalize=True)
+
+# Alternatively, you may also create PSDs from Epochs objects with psd_XXX
+f, ax = plt.subplots()
```

Comment:
I think methods on figures and axes are preferred to the global plotting
functions (e.g. plt.plot) in mpl